pennylane/labs/resource_estimation/resource_operator.py

```diff
@@ -95,5 +95,9 @@ def resource_rep_from_op(self) -> rc.CompressedResourceOp:
         return self.__class__.resource_rep(**params)
 
 
+class ResourceOperatorNotImplemented(Exception):
```

Comment:
Should add this to the resource estimation init file and the documentation module docstring.